cmd/template-resolver/utils/templateResolver.go

```diff
@@ -88,18 +92,20 @@ func ProcessTemplate(yamlBytes []byte, hubKubeConfigPath, clusterName string) ([
 		return nil, fmt.Errorf("failed to determine the kubeconfig to use: %w", err)
 	}
 
-	var hubResolver *templates.TemplateResolver
-
-	hubTemplateCtx := struct {
-		ManagedClusterName   string
-		ManagedClusterLabels map[string]string
-	}{ManagedClusterName: clusterName}
+	defaultHubTemplateOpts := &hubTemplateOptions{
+		config: templates.Config{
+			AdditionalIndentation: 8,
+			DisabledFunctions:     []string{},
+			StartDelim:            "{{hub",
+			StopDelim:             "hub}}",
+		},
+	}
 
-	var hubResolveOptions templates.ResolveOptions
+	var hubResolver *templates.TemplateResolver
 
 	if hubKubeConfigPath != "" {
 		if policy.GetNamespace() == "" {
```

Comment:
This doesn't apply to ConfigurationPolicy since the hub templates restriction is based on the root `Policy` namespace. Could you add a new argument for the user to supply the hub templates namespace? It'd fallback to the namespace if the input YAML is a `Policy` kind, but otherwise, it'd fail if it wasn't set.